### PR TITLE
Add Hotaru armor, another Japanese-Inspired Armor, with cleaner git

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

replace #13864

image

image

MAGENTA
image

image
## Why It's Good For The Game

More armor! Permission from ilikesaying but Griffer-249 to port this armor to TGMC.
## Changelog
:cl: novaepee, OrcaOrca the spriter
add: Add Hotaru armor, another Japanese-Inspired Armor! find it in the style line tab!
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]

**Comment by novaepee** [comment]
on god, i don't understand where the linter is coming from

File: code/modules/clothing/modular_armor/attachments/arm_plates.dm [hunk 1]
```diff
@@ -58,3 +58,8 @@
 	name = "\improper Style Pattern Kabuto arm plates"
 	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. All arm plates have the same armor and slowdown, meaning that only changing the chestplate makes significant armor difference. This armor appears to be marked as a Kabuto armor piece."
 	greyscale_config = /datum/greyscale_config/armor_mk1/kabuto
+
+obj/item/armor_module/armor/arms/marine/hotaru
```
Comment (hyper2snyper) [inline code/modules/clothing/modular_armor/attachments/arm_plates.dm:62]:
```suggestion
/obj/item/armor_module/armor/arms/marine/hotaru
```
This'll fix the linter

**Review by hyper2snyper — CHANGES_REQUESTED** [review]

**Comment by ilikesaying** [comment]
**_C O M P L E T E_**

**Review by ivanmixo — APPROVED** [review]
Meme

**Comment by Tailiik** [comment]
You can't attach the armor pieces to the xm-02 :hampter:

**Pull request merged**